src/validators.ts

```diff
@@ -50,6 +58,25 @@ class ValidatorPkg {
       validatorUri = this.validatorDir;
     }
 
+    // get the validator config file
+    const configFilePath = join(this.validatorDir, "config.json");
+    const configFile = readFileSync(configFilePath);
+    const validatorConfig = JSON.parse(configFile.toString());
+
+    // save the validator config state
+    ORIGINAL_VALIDATOR_CONFIG = JSON.stringify(validatorConfig, null, 2);
```

Comment:
@dtbuchholz This is left over from when this package only worked with docker.  What if we change this line to:
`ORIGINAL_VALIDATOR_CONFIG = configFile.toString();`
Then we don't have to deal with changing the lint and prettier setting for all of our ts packages?

Reply:
If that makes sense, we should probably do the same thing on lone 159 too

Reply:
@joewagner yes, nice solution! should be good to go now.